### DISCUSSION TITLE
$mail->embedFromPath(

### DISCUSSION THREAD

**Discussion by esisl** [pimcore/pimcore#10599] [Q&A / Support]
Looking at the documentation:
https://pimcore.com/docs/pimcore/current/Development_Documentation/Development_Tools_and_Details/Email_Framework/Pimcore_Mail.html

But if I try to do this in my code, I get:
[2021-10-14 11:37:29] request.CRITICAL: Uncaught PHP Exception Symfony\Component\ErrorHandler\Error\UndefinedMethodError: "Attempted to call an undefined method named "embedFromPath" of class "Pimcore\Mail".

What's wrong?

**Comment by dvesh3** [comment]
Are you using Pimcore X? this was introduced with v10.0

**Comment by esisl** [comment]
No, pimcore 6 :(
And in the sixth, how to do it?

**Reply by dvesh3** [reply]
Pimcore 6 Mail class extends `Swift_Message` from Swiftmailer. please check here how to embed images https://swiftmailer.symfony.com/docs/introduction.html

**Comment by esisl** [comment]
It's clear. Thanks
